vg/draw/canvas.go

```diff
@@ -26,14 +26,55 @@ type Canvas struct {
 }
 
 // TextStyle describes what text will look like.
+// Alignment and rotation are interrelated: when
+// (XAlign, YAlign) = (XCenter,YCenter), the text
+// is rotated about its center, (XLeft,YBottom)
+// causes the text to rotate around its bottom
+// left corner, etc.
 type TextStyle struct {
 	// Color is the text color.
 	Color color.Color
 
 	// Font is the font description.
 	Font vg.Font
+
+	// Rotation is the text rotation in radians.
+	// Positive rotation is counterclockwise.
+	Rotation float64
```

Comment:
Apologies for the reshift, but the text was troublesome. Here it could be (not perfect but clearer):

```
// Text rotation is performed around the axis
// defined by XAlign and YAlign.
```